test/e2e/specs/specs-playwright/wp-likes-spec.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+import {
+	BrowserHelper,
+	DataHelper,
+	LoginFlow,
+	NewPostFlow,
+	GutenbergEditorPage,
+	LikesComponent,
+} from '@automattic/calypso-e2e';
+
+/**
+ * Constants
+ */
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+const host = DataHelper.getJetpackHost();
+const viewportName = BrowserHelper.getViewportName();
+const quote =
+	'The foolish man seeks happiness in the distance. The wise grows it under his feet.\n— James Oppenheim';
+
+describe( `[${ host }] Likes: (${ viewportName }) @parallel`, function () {
+	this.timeout( mochaTimeOut );
+
+	describe( 'New post', function () {
+		let gEditor;
+		let likesComponent;
+
+		before( 'Can log in', async function () {
+			const loginFlow = new LoginFlow( this.page, 'gutenbergSimpleSiteUser' );
+			await loginFlow.login();
+		} );
+
+		it( 'Start new post', async function () {
+			const newPostFlow = new NewPostFlow( this.page );
+			await newPostFlow.newPostFromNavbar();
+		} );
+
+		it( 'Enter post title', async function () {
+			gEditor = await GutenbergEditorPage.Expect( this.page );
```

Comment:
Just a nitpick, but a strong one! 😄 I'd suggest a consistent naming for the instance objects. It's convenient to use shorter names but might bite us later on, when we have a gazillion different versions of it: `gEditor` / `gEditorPage` / `gutenbergEditor` / `gutenbergEditorPage` / etc. IMO it's just best to `snakeCase` the constructor and not even think about it: 
```js
const gutenbergEditorPage = await GutenbergEditorPage.Expect( this.page );
```

Maybe we can rename the `GutenbergEditorPage` to simply `GutenbergPage`? Are there any other pages we will be adding for Gutenberg?

Reply:
> Maybe we can rename the GutenbergEditorPage to simply GutenbergPage? Are there any other pages we will be adding for Gutenberg?

I would argue that we should maintain the naming of `GutenbergEditorPage` because it explicitly conveys that this page is an _Editor_ and it uses _Gutenberg_ for that purpose. It allows us to have additional editor types when the need arises.

Reply:
Renamed `gEditor` to the full `gutenbergEditorPage`.